components/canvas/NodeDescription.tsx

```diff
@@ -12,12 +12,12 @@ type NodeDescription = {
 export const NodeDescription = ({ header, description }: NodeDescription) => (
   <div className={styles["node__description-container"]}>
     {header && (
-      <Typography variant="caption" className={styles.header}>
+      <Typography variant="caption" className={styles["node__header"]}>
         {getNodeTypeName(header)}
       </Typography>
     )}
     {description && (
-      <Typography variant="caption" className={`${styles.description}`}>
+      <Typography variant="caption" className={styles["node__description"]}>
         {formatNodeText(description, 1000)}
```

Comment:
It doesn't seem like the limit of 1000 chars is doing its job on a card with room for 100 ish characters.

Reply:
The default value for the functions parameter is 100. This means that if the text consists of a lot of thin characters like "i" and "l" it will truncate the text even though it has not reached the end of the description box.